Src/SwqlStudio/Metadata/Entity.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace SwqlStudio.Metadata
 {
-    class Entity
+    public class Entity
```

Comment:
It doesn't really matter, but what is the reason for making this public? Same with `Property` below.

Reply:
I used same schema for creating tab as in Verbs -> method in `public interface IApplicationService`. That method receives Entity, thus requires the Entity to be public.